src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+
+* User type and assigned roles can affect your NerdGraph permissions. For more details, see [Factors affecting access](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/#user-permissions).
+
+## Mutation operations
+
+To enable users to interact with the Secrets Management Service, following are the exposed four NerdGraph mutations:
+
+
+### Create a secret[#create-secret]
+
+The `secretsManagementCreateSecret` mutation allows you to add new secrets to the Secrets Management Service.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `alias` (String, optional): An alias for the secret.
+* `entityGuid` (String, optional): The entity GUID associated with the secret.
+* `key` (String, required): The key for the secret.
+* `namespace` (String, required): The namespace to which the secret belongs.
+* `value` (SecureValue, required): The value of the secret.
+
+#### Response
+
+* `createdAt` (String): The time the secret was created.
+* `updatedAt` (String): The time the secret was last updated.
+* `key` (String): The key of the created secret.
+* `latestVersion` (String): The version of the created secret.
+* `tags` (Array): Metadata associated with the secret.
+
+#### Sample query:
+
+```graphql
+mutation {
+  secretsManagementCreateSecret(
+    accountId: 12345678
+    key: "test_user_key"
+    value: "1234567890"
+    alias: ""
+    entityGuid: ""
+    namespace: ""
+  ) {
+    createdAt
+    latestVersion
+    updatedAt
+  }
+}
+
+```
+#### Smaple response:
+    
+    ```json
+    {
+    "data": {
+        "secretsManagementCreateSecret": {
+        "createdAt": "2021-09-29T10:00:00Z",
+        "latestVersion": "1",
+        "updatedAt": "2021-09-29T10:00:00Z"
+        }
+    }
+    }
+    ```
+
+
+### Purge secrets[#purge-secrets]
+
+The `secretsManagementPurgeSecret` mutation allows you to delete secrets from the Secrets Management Service. You can delete a specific version of a secret or all versions of a secret.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `key` (String, required): The key for the secret.
+* `namespace` (String, optional): The namespace to which the secret belongs.
+* `purgeAll` (Boolean, required): If true, all versions of the secret are deleted. 
+* `version` (String, optional): The version of the secret to delete.
+
+<Callout variant="tip">
+    * If `purgeAll` is true, the `version` parameter is null and ignored.
+    * valid combination of `purgeAll` and `version` : `true` & `null`, `false` & `non-null`.
+</Callout>
+
+#### Response
+
+* `key` (String): The key of the created secret.
+* `latestVersion` (String): The latest version available corresponding to the key. If `purgeAll` is true or no version is available after purge, latestVersion will be null.
+* `tags` (Array): Metadata associated with the secret.
```

Comment:
I dont see key and tags in the sample response provided. Am I looking at the right copy, coz you told me you'll update the responses and share the draft...

Again, let's use 'meaningful' samples in our docs.